sdk/documentintelligence/azure-ai-documentintelligence/src/samples/README.md

```diff
@@ -0,0 +1,110 @@
+---
+page_type: sample
+languages:
+  - java
+products:
+  - azure
+  - azure-cognitive-services
+  - azure-form-recognizer
```

Comment:
If we don't use form recognizer terms anymore. Search for the term `azure-form-recognizer` and `form recognizer` and replace them with document intelligence.

Reply:
I removed all the text appearances of "form recognizer". You may find some references in links, vteam still needs to port those to new brand name so will use them as-is for now.